package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "soap-component-oih",
+  "version": "1.0.0",
+  "description": "This is the soap api component for OIH",
+  "main": "node_modules/@openintegrationhub/ferryman/runGlobal.js",
+  "scripts": {
+    "compile": "eslint . --ext .ts && tsc",
+    "test": "LOG_LEVEL=fatal ts-mocha test --recursive --timeout 50000",
+    "test:debug": "LOG_LEVEL=trace DEBUG=nock.* ts-mocha test --recursive --timeout 50000"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/blendededge/soap-component-oih.git"
+  },
+  "keywords": [
+    "SOAP",
+    "OIH",
+    "Connector"
+  ],
+  "author": "Blended Edge",
+  "license": "Apache-2.0",
+  "bugs": {
+    "url": "https://github.com/blendededge/soap-component-oih/issues"
+  },
+  "homepage": "https://github.com/blendededge/soap-component-oih#readme",
+  "dependencies": {
+    "@openintegrationhub/ferryman": "^2.1.0",
+    "axios": "^0.25.0",
+    "soap": "^0.43.0"
+  },
+  "devDependencies": {
+    "@elastic.io/component-logger": "^0.0.1",
+    "@types/node": "^17.0.10",
+    "@typescript-eslint/eslint-plugin": "^5.10.0",
+    "@typescript-eslint/parser": "^5.10.0",
+    "chai": "^4.3.4",
+    "eslint": "^8.7.0",
+    "mocha": "^9.2.0",
+    "nock": "^13.2.2",
+    "testdouble": "^3.16.4",
+    "typescript": "^4.5.5",
+    "yarn": "^1.22.17"
```

Comment:
I would like to standardize on npm since the latest features are on par with yarn